tests/unittests/GraphOptzTest.cpp

```diff
@@ -659,54 +689,69 @@ TEST_F(GraphOptz, sinkTransposeBelowRescale) {
   EXPECT_EQ(convTRInput.dims(), llvm::makeArrayRef(origDims));
   EXPECT_EQ(convTRInput.getNode()->getNthInput(0).dims(),
             llvm::makeArrayRef(origDims));
-  EXPECT_EQ(F_->getNodes().size(), 3);
+  EXPECT_EQ(originalF_->getNodes().size(), 3);
 }
 
 TEST_F(GraphOptz, cancelTwoTransposes) {
   const size_t origDims[] = {1, 5, 10, 15};
-  Node *A = mod_.createPlaceholder(ElemKind::FloatTy, origDims, "input", false);
-  Node *T1 = F_->createTranspose("transpose", A, NCHW2NHWC);
-  Node *T2 = F_->createTranspose("transpose", T1, NHWC2NCHW);
-  ReluNode *K = F_->createRELU("relu", T2);
-  SaveNode *save = F_->createSave("ret", K);
+  Placeholder *A =
+      mod_->createPlaceholder(ElemKind::FloatTy, origDims, "input", false);
+  Node *T1 = originalF_->createTranspose("transpose", A, NCHW2NHWC);
+  Node *T2 = originalF_->createTranspose("transpose", T1, NHWC2NCHW);
+  ReluNode *K = originalF_->createRELU("relu", T2);
+  SaveNode *save = originalF_->createSave("ret", K);
 
   EXPECT_EQ(K->getInput().dims(), llvm::makeArrayRef(origDims));
-  EXPECT_EQ(F_->getNodes().size(), 4);
+  EXPECT_EQ(originalF_->getNodes().size(), 4);
+
+  optimizedF_ = originalF_->clone(originalF_->getName().str() + "_optmized");
```

Comment:
These two lines are going to be the same for all tests. They can be moved into a small helper function.